### ISSUE TITLE
Unexpected warning from v4l2

### ISSUE THREAD

**Issue by waikeat2397** [raspberrypi/picamera2#1066]
No sure we should be aware of this warning message or not. I saw this warning message in my setup.

Setup
Linux raspberry 6.6.20+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.20-1+rpt1 (2024-03-07) aarch64 GNU/Linux
Raspberry Pi 5
Raspberry Pi Camera Module 3
libcamera v0.3.0+65-6ddd79b5
libpisp version v1.0.6 b567f0455680 17-06-2024
picamera2 0.3.19

![Screenshot 2024-06-28 231012](https://github.com/raspberrypi/picamera2/assets/122470067/523d7044-5850-4263-afd0-4b62e0794663)

**Comment by davidplowman** [comment]
Yes, that's normal, libcamera can be a bit "chatty" sometimes. You can quieten it down by putting something like
```
LIBCAMERA_LOG_LVELS=*:3
```
in your `.bashrc` or typing it in the terminal window.
